FF152 Relnote TC39 Iterator includes proposal (#44258)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

## JavaScript

### TC39 Iterator includes proposal

> Support for the [`Iterator.prototype.includes()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/includes) method, which allows testing of whether an `Iterator` instance will return a specified value.

Reviewer: Or: 

```suggestion
The [`Iterator.prototype.includes()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/includes) method tests whether an `Iterator` instance will produce a specified value.
```

or if you wish "Support for....method has been added, which.." to make the fragment a complete sentence
